spec/index.html

```diff
@@ -578,33 +618,36 @@ <h4>
               console.log(reader.result);
           };
           reader.readAsText(file);
-        </pre> 
-      </section>     
-    </section>
-    
-    <section id='communication'>
-      <h3>
-      <span its-locale-filter-list="en" lang="en">Communication between Widget and MiniApp</span>
-      <span its-locale-filter-list="zh-hans" lang="zh-hans">Widget和MiniApp间的通讯</span>
-      </h3>
-      
-      <p its-locale-filter-list="zh-hans" lang="zh-hans">Widget 和 MiniApp 之间能够通过一系列 API 进行通信。这会涉及 MiniApp 的后台运行和service相关概念。</p>
-      <p its-locale-filter-list="en" lang="en">Widget and MiniApp can communicate through a series of APIs. This will involve MiniApp's background running and service concepts.</p>
-    </section>
-  </section>
+        </pre> 		
+      </section>
+	
+		
 
   <section>
     <h2>
      <span its-locale-filter-list="en" lang="en">Security considerations</span>
       <span its-locale-filter-list="zh-hans" lang="zh-hans">安全考虑</span>  
       </h2>
+	    <p its-locale-filter-list="zh-hans" lang="zh-hans">由于widget的packaging、addressing和manifest与MiniApp page类似，因此widget的安全考虑宜参考MiniApp packaging、addressing和manifest中的安全考虑。此外：</p>
+    <p its-locale-filter-list="en" lang="en">Since the packaging, addressing, and manifest of widgets are similar to MiniApp pages, the security considerations of widgets should refer to the security considerations in MiniApp packaging, addressing, and manifest. Furthermore:</p>
```

Comment:
```suggestion
	    <p its-locale-filter-list="zh-hans" lang="zh-hans">由于widget的packaging、addressing和manifest与MiniApp page类似，因此widget的安全考虑宜参考MiniApp <a href="https://www.w3.org/TR/miniapp-packaging/#sec-security">Packaging</a>、<a href="https://www.w3.org/TR/miniapp-addressing/#security-considerations">Addressing</a>和<a href="https://www.w3.org/TR/miniapp-manifest/#security-considerations">Manifest</a>中的安全考虑。此外：</p>
    <p its-locale-filter-list="en" lang="en">Since the packaging, addressing, and manifest of widgets are similar to MiniApp pages, the security considerations of widgets should refer to the security considerations in MiniApp <a href="https://www.w3.org/TR/miniapp-packaging/#sec-security">Packaging</a>, <a href="https://www.w3.org/TR/miniapp-addressing/#security-considerations">Addressing</a>, and <a href="https://www.w3.org/TR/miniapp-manifest/#security-considerations">Manifest</a>. Furthermore:</p>
```